### PR TITLE
[tensorflow][ecs][training]Change ecs training model

### PR DESCRIPTION
**Pull request by Teng-xu**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

#### EIA Checklist
* When creating a PR:
- [x] I've modified ```src/config/build_config.py``` in my PR branch by setting ```ENABLE_EI_MODE = True```
* When PR is reviewed and ready to be merged:
- [x] I've reverted the code change on the config file mentioned above

* For reviewer, before merging, please cross-check:
- [x] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Change ecs tensorflow training model to Faster RCNN
The Faster RCNN model is built from deep learning models repo:
https://github.com/aws-samples/deep-learning-models/blob/master/models/vision/detection/docs/FASTER_RCNN.md

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by YYStreet** [comment]
Is this PR ready for review? Are we preferring to add the model darknet back?

**Comment by Teng-xu** [comment]
> Is this PR ready for review? Are we preferring to add the model darknet back?
Yes, this pr is ready for review, and for now we decided to not add darknet back.

**Comment by TusharKanekiDey** [comment]
Can you add relevant papers and article about the models which are needed in this PR in the description?

**Comment by Teng-xu** [comment]
> Can you add relevant papers and article about the models which are needed in this PR in the description?

Yep, added the link to the model

**Comment by TusharKanekiDey** [comment]
> Just to confirm, do we not need to test mnist on ecs afterwards?

I think we should skip mnist if we are having Faster RCNN for ECS.

**Comment by Teng-xu** [comment]
> > Just to confirm, do we not need to test mnist on ecs afterwards?
> 
> I think we should skip mnist if we are having Faster RCNN for ECS.

Yeah, we can use Faster RCNN for ECS and skip mnist

**Review by YYStreet — DISMISSED** [review]

**Comment by nskool** [comment]
Ping - what's the update on the PR?

**Comment by tejaschumbalkar** [comment]
Waiting for the PR test builds to pass

**Review by tejaschumbalkar — APPROVED** [review]

**Comment by tejaschumbalkar** [comment]
Test failures due to resource unavailability

File: test/dlc_tests/ecs/tensorflow/training/test_ecs_tensorflow_training.py [hunk 1]
```diff
@@ -29,16 +30,16 @@ def test_ecs_tensorflow_training_mnist_cpu(cpu_only, ecs_container_instance, ten
     ecs_utils.ecs_training_test_executor(ecs_cluster_name, cluster_arn, training_cmd, tensorflow_training, instance_id)
 
 
-@pytest.mark.model("mnist")
-@pytest.mark.parametrize("training_script", [TF_MNIST_TRAINING_SCRIPT], indirect=True)
+@pytest.mark.model("FasterRCNN")
+@pytest.mark.parametrize("training_script", [TF_FasterRCNN_TRAINING_SCRIPT], indirect=True)
 @pytest.mark.parametrize("ecs_instance_type", ["p2.8xlarge"], indirect=True)
```
Comment (saimidu) [inline test/dlc_tests/ecs/tensorflow/training/test_ecs_tensorflow_training.py:35]: Does this test need a p2.8xlarge with 8 GPUs? Can it use a smaller instance type or a smaller number of GPUs?

Comment (Teng-xu) [reply test/dlc_tests/ecs/tensorflow/training/test_ecs_tensorflow_training.py:35]: Yeah, but default the instance type is p2.8xlarge, and I didn't change that, I think we should keep the same with the other tests

File: test/dlc_tests/container_tests/bin/testFasterRCNN [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pip install s3fs
+pip install addict
+pip install tensorflow_addons
+
+git clone https://github.com/aws-samples/deep-learning-models.git
+wget https://github.com/keras-team/keras-applications/releases/download/resnet/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 -P /deep-learning-models/models/vision/detection/weights/
+mv ${BIN_DIR}/faster_rcnn_r50_fpn_1x_coco.py /deep-learning-models/models/vision/detection/configs/faster_rcnn/EC2/
+
+cd deep-learning-models/models/vision/detection/
+scripts/train_ec2_single_node.sh 8 /deep-learning-models/models/vision/detection/configs/faster_rcnn/EC2/faster_rcnn_r50_fpn_1x_coco.py
```
Comment (saimidu) [inline test/dlc_tests/container_tests/bin/testFasterRCNN:33]: Please check for the exit code of the script, and exit this script with that code.

**Review by SergTogul — APPROVED** [review]

**Pull request merged**